### PR TITLE
[Pibbble][#102991688]Landing page design

### PR DESCRIPTION
**Pull request by wilmkr**
I have squashed my commits

### CODE REVIEW

**Comment by unicodeveloper** [comment]
Take of the StyleCI issues also and rebase to resolve the conflicts.

File: public/css/style.css [hunk 1]
```diff
@@ -167,4 +167,8 @@ a:hover {
 
 .blank-page {
     min-height: 450px;
+<<<<<<< HEAD
 }
+=======
+}
+>>>>>>> c8f19a3... [Pibbble][#102991688]Landing page design
```
Comment (unicodeveloper) [inline public/css/style.css]: You committed merge conflicts?. :flushed: Please resolve

File: resources/views/landing.blade.php [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+      echo("<a href=''>");
+      echo("<img src='/img/shot.png' width='200' height='150' style='border:0px solid #ccc;' />");
+      echo("</a>");
+      echo("<span class='project-stats'><i class='fa fa-thumbs-o-up'></i>&nbsp;0</span>");
+      echo("<span class='project-stats'><i class='fa fa-eye'></i>&nbsp;0</span>");
+      echo("</div>");
+      echo("<span class='projects-name'>Project title</span>");
+      echo("</div>");
+    }
+    ?>
```
Comment (unicodeveloper) [inline resources/views/landing.blade.php]: Wow, why do we have this kind of code here in the view? I understand that this is meant to be a placeholder for the contents of the page right? Use [blade's](http://laravel.com/docs/5.1/blade) syntax instead. Then abstract it into another view and just include it here using `@include`

**Comment by unicodeveloper** [comment]
StyleCI is failing, please fix ASAP!

**Comment by wilmkr** [comment]
Rectified the style CI

**Comment by unicodeveloper** [comment]
Alright cool, now rebase

**Pull request merged**